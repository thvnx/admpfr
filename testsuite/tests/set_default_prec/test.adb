with Ada.Text_IO; use Ada.Text_IO;
with Admpfr;      use Admpfr;

procedure Test is
   M, N : Mpfloat;
begin
   Set (M, "1");
   Set_Default_Prec (42);
   Set (N, "1");
   Put_Line (Get_Prec (M)'Image);
   Put_Line (Get_Prec (N)'Image);

   declare
      O : Mpfloat;
   begin
      Put_Line (Get_Prec (O)'Image);
   end;
end Test;
