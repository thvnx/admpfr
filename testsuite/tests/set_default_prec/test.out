 53
 53
 42
